src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1346,6 +1321,32 @@ private void UpdateSymmetryIcon()
         }
     }
 
+    /// <summary>
+    ///   Lock / unlock a single organelle that need a nucleus
+    /// </summary>
```

Comment:
isn't that summary wrong? Organelles that don't need a nucleus also get fed into this method.

Reply:
In the future this might be changed as the method get more advanced, but for now I think this makes sense as what the method basically does is what the summary says.